articles/quickstart/webapp/aspnet-core/_includes/_login.md

```diff
@@ -366,7 +353,7 @@ public void ConfigureServices(IServiceCollection services)
 }
 ```
 
-To retrieve the tokens, you can call `GetTokenAsync`:
+To retrieve the tokens, you can call `GetTokenAsync` and use it as needed:
```

Comment:
```suggestion
To retrieve the tokens, call `GetTokenAsync` and use them as required:
```

Reply:
I made the exact change in the ASPNET Core 3.1 quickstart as part of this PR.